packages/portals/frontend/sirius-components-portals/src/hooks/usePortalMutations.types.ts

```diff
@@ -11,6 +11,8 @@
  *     Obeo - initial API and implementation
  *******************************************************************************/
 
+import { GQLPayload } from '@eclipse-sirius/sirius-components-core';
```

Comment:
This change is a bit more wrong than I believed since in this case, the typing provided does not match at all what is received from the GraphQL API since not a single one of the mutation here retrieves the `messages` of the success payload. The field messages in what was your `GQLPayload` should at the very least be nullable since people can forget to ask for the field in question.